backend/rust/Cargo.lock

```diff
@@ -0,0 +1,1333 @@
+# This file is automatically @generated by Cargo.
```

Comment:
You can put in gitignore Cargo.lock for the time being.

Reply:
Done